README.md

```diff
@@ -135,6 +135,53 @@ Parcelgen supports writing and reading the following types to a Parcel:
 }
 ```
 
+### Using Enums
+
+
+Parcelgen provides full support for working with enums. You can add enum fields to your models following the next steps:
```

Comment:
I did it from this shitty interface and it looks it didn't get pushed, I'll push it again